### DISCUSSION TITLE
关于“场景3-推理”的答案

### DISCUSSION THREAD

**Discussion by kamaliang** [thinkingjimmy/Learning-Prompt#49] [Q&A] [answered]
答案是A，而不是D。
另外我在今天（4月1日）的chatGPT里问同样的问题，它给的答案也是A

**Comment by thinkingjimmy** [comment, marked as answer]
我刚又去问了下 ChatGPT，你用的是 API 版本？或者是非 ChatGPT 的官方 IM ？如果是 API 版本，大概率是 A。
<img width="623" alt="image" src="https://user-images.githubusercontent.com/37492595/230064224-0c6feaab-efe2-4128-b60f-42aede564553.png">
